templates/fsharp/app-mvvm/App.axaml.fs

```diff
@@ -0,0 +1,20 @@
+namespace AvaloniaAppTemplate
+
+open Avalonia
+open Avalonia.Controls.ApplicationLifetimes
+open Avalonia.Markup.Xaml
+open AvaloniaAppTemplate.ViewModels
+open AvaloniaAppTemplate.Views
+
+type App() as self =
+    inherit Application()
+
+    do AvaloniaXamlLoader.Load self
+
+    override x.OnFrameworkInitializationCompleted() =
+        match x.ApplicationLifetime with
+        | :? IClassicDesktopStyleApplicationLifetime as desktop ->
+             desktop.MainWindow <- new MainWindow(DataContext=MainWindowViewModel())
+        | _ -> ()
```

Comment:
```suggestion
    override this.OnFrameworkInitializationCompleted() =
        match this.ApplicationLifetime with
        | :? IClassicDesktopStyleApplicationLifetime as desktop ->
             desktop.MainWindow <- new MainWindow(DataContext = MainWindowViewModel())
        | _ -> ()
```

```suggestion
    override self.OnFrameworkInitializationCompleted() =
        match self.ApplicationLifetime with
        | :? IClassicDesktopStyleApplicationLifetime as desktop ->
             desktop.MainWindow <- new MainWindow(DataContext = MainWindowViewModel())
        | _ -> ()
```

Reply:
it's against F# style guide:

https://docs.microsoft.com/en-us/dotnet/fsharp/style-guide/formatting#formatting-white-space-in-expressions

Reply:
Ok, I won't apply this one if it's against the style guide.